free-1.md

```diff
@@ -57,12 +155,31 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Honestly, a lot of this work would focus on the "systems" aspects because 
+dealing with audio is hard. And I'm not sure how this DSL would be implemented. 
```

Comment:
Java has a pretty comprehensive Midi library (presumably also available with Scala) so I think as long as the host language has decent tooling you'll be okay.